cmd/coronadecode/coronadecode.go

```diff
@@ -26,6 +26,7 @@ func printCertificate(decoded *coronaqr.Decoded) {
 func main() {
 	var (
 		verify    = flag.Bool("verify", false, "verify the signature in addition to decoding")
+		expiredOK = flag.Bool("expiredOK", false, "don't abort program when wanting to inspect an expired certificate")
```

Comment:
The flag is named expiredOK, but the code prints the certificate regardless of whether the expiration is the problem, or something else.

Also, expiredOK sounds too harmless IMO. A more descriptive name would be `print_expired`.

But perhaps the flag should be called `insecure` given that it prints despite verification errors?